File: programmers/중복된 문자 제거.js
// 문자열 my_string이 매개변수로 주어집니다. my_string에서 중복된 문자를 제거하고 하나의 문자만 남긴 문자열을 return

function solution(my_string) {
  let set = new Set();
  let new_string = "";

  for (let alpha of my_string) {
    set.add(alpha);
  }

  for (let alpha of set) {
    new_string += alpha;
  }

  return new_string;
}

console.log(solution("We are the world") === "We arthwold");

File: programmers/python/중복된 문자 제거.py
# 중복된 문자 제거
def solution(my_string):
    answer = []
    for i in my_string :
        if answer.count(i) == 0 :
            answer.append(i)
    return ''.join(answer)


def solution(my_string):
    return ''.join(dict.fromkeys(my_string))


def solution(my_string):
    answer = ''
    for i in my_string:
        if i not in answer:
            answer+=i
    return answer